README.md

```diff
@@ -454,6 +456,40 @@ Usage: `akamai show-restier <edgeworkerId>`
 | - | - | - |
 | edgeworkerId | required | Edgeworker identifier.
 
+### Get Available EdgeWorkers Report Types
+Allows customers to list the avaiable report types that can be generated for an EdgeWorker ID.
+
+Usage: `akamai get reports`
+
+| Option | Description |
+| - | - |
+| -h, --help  | output usage information |
+
+#### Key Details
+1. The user will be given a table with reportIds and a description of each report. Use a given reportId with the `get report` command to get a report for a given EdgeWorker.
+
+### Get Edgeworker Report
+Allows customers to get a report for a given EdgeWorker ID.
+
+Usage: `akamai get report <reportId> <edgeworker-identifier>`
+
+| Option | Existence| Description |
+| - | - | - |
+| -h, --help  | optional | output usage information |
+| -s, --startDate `<startDate>` | required | ISO 8601 timestamp indicating the start time of the EdgeWorker report. |
+| -e, --endDate `<startDate>` | optional | ISO 8601 timestamp indicating the end time of the EdgeWorker report. If not specified, the end time defaults to the current time. |
+| --status `<status>` | optional | Comma-separated string to filter by EdgeWorker status. Values: `success`, `genericError`, `unknownEdgeWorkerId`, `unimplementedEventHandler`, `runtimeError`, `executionError`, `timeoutError`, `resourceLimitHit`, `cpuTimeoutError`, `wallTimeoutError`, `initCpuTimeoutError`, `initWallTimeoutError`. |
+| --ev, --eventHandlers `<eventHandlers>` | optional | Comma-separated string to filter EdgeWorkers by the event that triggers them. Values: `onClientRequest`, `onOriginRequest`, `onOriginResponse`, `onClientResponse`, `responseProvider`. |
+
+| Argument | Existence | Description |
+| - | - | - |
+| reportId | required | Report Type. |
+| edgeworker-identifier | required | Edgeworker identifier. |
+
+#### Key Details
+1. For a list of available report IDs, use the `get reports` command.
+2. The `startDate` option is a required option.
+
```

Comment:
can you get this reviewed by sarah kotyck. She usually reviews them and also let her know the newly supported commands/changes so she can add it in her release notes